### PR TITLE
ci: translation-advice: avoid commenting after make i18n-extract [AUTH-145]

### PR DESCRIPTION
**Pull request by rissson**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: .github/workflows/translation-advice.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ on:
     paths:
       - "!**"
       - "locale/**"
+      - "!locale/en/**"
       - "web/src/locales/**"
+      - "!web/src/locales/en.ts"
```
Comment (rissson) [inline .github/workflows/translation-advice.yml:12]: Do we need to add pseudo-LOCALE.ts here as well?

Comment (BeryJu) [reply .github/workflows/translation-advice.yml:12]: All of the web/src locales are generated at build time: https://github.com/goauthentik/authentik/tree/main/web/src/locales

Comment (rissson) [reply .github/workflows/translation-advice.yml:12]: Ah, so we can entirely remove the web stuff in here.

For my own comprehension, does the build download them from transifex then?

**Comment by rissson** [comment]
Ok this only works for backend files, as all frontend files are updated when source strings are modified

File: .github/workflows/translation-advice.yml (hunk 1 above)
Comment (BeryJu) [reply .github/workflows/translation-advice.yml]: Nothing is downloaded, the translation source files are in GitHub and get converted to typescript during build, anytime anything changes in transifex it gets PR'd to the repo

**Review by BeryJu — APPROVED** [review]

**Pull request merged**